mmyolo/deploy/codebase/mmyolo/models/dense_heads/yolov5_head.py

```diff
@@ -0,0 +1,109 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import copy
+from typing import List, Optional, Tuple
+
+import torch
+from mmengine.config import ConfigDict
+from mmengine.structures import InstanceData
+from torch import Tensor
+
+from mmdeploy.codebase.mmdet import get_post_processing_params
+from mmdeploy.codebase.mmdet.models.layers import multiclass_nms
+from mmdeploy.core import FUNCTION_REWRITER
+
+
+@FUNCTION_REWRITER.register_rewriter(
+    func_name='mmyolo.models.dense_heads.yolov5_head.'
+    'YOLOv5Head.predict_by_feat')
+def yolov5_head__predict_by_feat(ctx,
```

Comment:
```suggestion
def yolov5_head_predict_by_feat(ctx,
```

Reply:
> 

In mmdeploy, we define the rewriter function name as: `model__function__backend`, so I think `yolov5_head__predict_by_feat` is okay.